docs/root/intro/arch_overview/security/rbac_filter.rst

```diff
@@ -99,6 +99,7 @@ The following attributes are exposed to the language runtime:
    connection.dns_san_peer_certificate, string, The first DNS entry in the SAN field of the peer certificate in the downstream TLS connection
    connection.uri_san_local_certificate, string, The first URI entry in the SAN field of the local certificate in the downstream TLS connection
    connection.uri_san_peer_certificate, string, The first URI entry in the SAN field of the peer certificate in the downstream TLS connection
+   connection.id, uint, Downstream connection ID
```

Comment:
nit: is it accurate that it is 0 if not set? (and in the access log doc)

Reply:
No, it becomes "unset", which is an error. `has(connection.id)` would return `false`. For access log, it looks it's reasonable to return 0 since I'm not sure how missing values are handled there.

Reply:
Ah, gotcha. Yeah, for access logs the extractor needs a 0 value, like response codes setting 0 for unset values.